src/App/Pages/Generator/GeneratorHistoryPageViewModel.cs

```diff
@@ -58,8 +58,11 @@ public async Task ClearAsync()
         private async void CopyAsync(GeneratedPasswordHistory ph)
         {
             await _clipboardService.CopyTextAsync(ph.Password);
-            _platformUtilsService.ShowToast("info", null,
+            if (!_clipboardService.CopyNotificationHandledByOs())
+            {
+                _platformUtilsService.ShowToast("info", null,
                 string.Format(AppResources.ValueHasBeenCopied, AppResources.Password));
+            }
```

Comment:
Given that this is becoming a boilerplate and being used quite often, we could make an extension method for this:
```c#
public static void ShowToastForCopiedValue(this IPlatformUtilsService platformUtilsService, IClipboardService clipboardService, string valueNameCopied)
{
    if (!clipboardService.CopyNotificationHandledByOs())
    {
        platformUtilsService.ShowToast("info", null, string.Format(AppResources.ValueHasBeenCopied, valueNameCopied));
    }
}
```
So we can just use:
```c#
_platformUtilsService.ShowToastForCopiedValue(_clipboardService, AppResources.Password);
```
Or the other way is just to add the method to `IPlatformUtilsService`